daemon/cmd/daemon_main.go

```diff
@@ -1370,6 +1373,24 @@ func initEnv(cmd *cobra.Command) {
 			option.EgressMultiHomeIPRuleCompat,
 		)
 	}
+
+	if option.Config.InstallNoConntrackIptRules {
+		// InstallNoConntrackIptRules can only be enabled in direct
+		// routing mode as in tunneling mode the encapsulated traffic is
+		// already skipping netfilter conntrack.
+		if option.Config.Tunnel != option.TunnelDisabled {
+			option.Config.InstallNoConntrackIptRules = false
+			log.Fatalf("%s requires the agent to run in direct routing mode.", option.InstallNoConntrackIptRules)
```

Comment:
nit: if we fatal here, why the preceeding `option.Config.InstallNoConntrackIptRules = false`?

Reply:
Leftovers as before I was just emitting a warning, let me remove them :+1: